DXRModules/DeusEx/Classes/DXRFlags.uc

```diff
@@ -806,6 +815,8 @@ function string GameModeName(int gamemode)
         return "Halloween Mode";// maybe needs a better name
     case OneItemMode:
         return "One Item Mode";
+    case BingoCampaign:
```

Comment:
This should probably be wrapped with #ifdef vanilla until it's tested and polished in other mods

Reply:
I don't understand. When would there be a case where something is asking for the mode name and you don't want to give it an answer?

Reply:
When the mode isn't supported in other mods, like Entrance Rando

Reply:
Why would they ask the name of the mode though?